packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.9.0] - 2022-09-07
+
+### Changed
+- Sync with Substrate SDK to include all latest features like:
+  - Worker threads.
```

Comment:
@stwiname maybe we could elaborate on `worker threads` and `bulk operations` to highlight value?

Reply:
Maybe we could leave a comment to see the main sdk changelog?

Reply:
Perfect!